docs/DeveloperGuide.md

```diff
@@ -262,27 +262,50 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
+A sales and customer relations representative working in the F&B industry. In
+particular, this representative works with B2B sales.
+
+* has a need to manage a significant number of business contacts
 * prefer desktop apps over other types
 * can type fast
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
+* has a need to transfer large number of contacts to new workers
```

Comment:
Not sure if this should be part of user profile.

Reply:
I added this incorrectly. I was intending to add this to justify for the user story below about "export a list of contact emails".

Will remove.